### PR TITLE
Update Dive into deep learning book

### PR DESCRIPTION
**Pull request by xeroxism**
Added authors and PDF link to Dive into deep learning book.

## Hacktoberfest notes:

- due to volume of submissions, we may not be able to review PRs that do not pass tests and do not have informative titles.
- please read our [contributing guidelines](/CONTRIBUTING.md)
- be sure to check the output of Travis-CI for linter errors
- if this is your first open source contribution, make sure it's not your last!

## What does this PR do?
Add Resource(s) | Remove Resource(s) | Add info | Improve Repo

### Checklist:
- [ ] Not a duplicate
- [ ] Included author(s) if appropriate
- [ ] Lists are in alphabetical order
- [ ] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

File: free-programming-books.md [hunk 1]
```diff
@@ -471,7 +471,7 @@
 * [Bayesian Reasoning and Machine Learning](http://web4.cs.ucl.ac.uk/staff/D.Barber/pmwiki/pmwiki.php?n=Brml.HomePage)
 * [Deep Learning](http://www.deeplearningbook.org) - Ian Goodfellow, Yoshua Bengio and Aaron Courville
 * [Deep Learning with PyTorch](https://pytorch.org/assets/deep-learning/Deep-Learning-with-PyTorch.pdf) - Eli Stevens, Luca Antiga, Thomas Viehmann (PDF)
-* [Dive into Deep Learning](http://d2l.ai)
+* [Dive into Deep Learning](http://d2l.ai) - Aston Zhang et al (HTML) [PDF](https://d2l.ai/d2l-en.pdf)
```
Comment (eshellman) [inline free-programming-books.md:474]: good to add creators, but we don't usually list seperate links for particular formats

Comment (xeroxism) [reply free-programming-books.md:474]: OK. 

I saw a format like that though.

Comment (eshellman) [reply free-programming-books.md:474]: Occasionally there's a good reason. not here.

**Comment by SethiPandi** [comment]
Closing this as the resource is already in the repository.

**Pull request closed without merging**